sklearn/semi_supervised/_label_propagation.py

```diff
@@ -97,6 +97,15 @@ class BaseLabelPropagation(ClassifierMixin, BaseEstimator, metaclass=ABCMeta):
      tol : float, default=1e-3
          Convergence tolerance: threshold to consider the system at steady
          state.
+     
+     unlabeled_mark: {-1, 'unk'} or custom value, default=-1
```

Comment:
I assume that it would be better to accept a list with the potential value to be considered as unlabeled.
It looks like something similar to what we are using for the detection of the missing value.

We can also accept a single item that we can wrap in a list.